src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,22 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
+        return valuesCount > 1
+    }
+
+
+    ProtoValue getValue() {
+        innerProtoProperty.valuesList.first()
```

Comment:
@jbornemann Looks like there's some cases where there are no values present on the node which causes the overall job to fail.

29.07.2016 17:15:12.973 *ERROR* [clientJobLauncherTaskExecutor-9] org.springfram
ework.batch.core.step.AbstractStep Encountered an error executing step clientJcr
Nodes in job clientJob
java.util.NoSuchElementException: Cannot access first() element from an empty Li
st
        at org.codehaus.groovy.runtime.DefaultGroovyMethods.first(DefaultGroovyM
ethods.java:7010)
        at com.twcable.grabbit.jcr.ProtoPropertyDecorator.getValue(ProtoProperty
Decorator.groovy:93)
        at com.twcable.grabbit.jcr.ProtoPropertyDecorator.getPropertyValue(Proto
PropertyDecorator.groovy:98)